### ISSUE TITLE
HTML tags in markdown are being stripped

### ISSUE THREAD

**Issue by jaydreyer** [Hypercubed/angular-marked#69]
Not sure what to do. I have html code samples in my markdown file within ```html <code> ``` type structure. When the markdown is rendered the html tags are stripped out of the examples. Any ideas? Thanks.

**Comment by Hypercubed** [comment]
Can you give an example?  Are you using the markdown backticks?

**Comment by jaydreyer** [comment]
I'm basically using the file below. I had to add .txt to it so that I could upload it. I can get things to work if I escape all < characters. I'm not sure if that is standard practice but I saw you did that in your code example so I tried it and it workd.

Yes, I'm using the ``` backticks.

[markdown.md.txt](https://github.com/Hypercubed/angular-marked/files/606303/markdown.md.txt)

**Comment by jaydreyer** [comment]
Ok. I think I figured it out. I'm posting what I did in case it helps others.  We had another package in our app that was using a previous version of highlight.js, and for whatever reason, it kept winning out. When I removed the old version and replaced with the latest, this started working, and the old stuff still worked fine. I did about 100 other things, but I'm confident this was what made it ultimately work. I then removed all of the escaping from my markdown file and things worked perfectly. Now I just have to figure out how to get the agate.css style to override whatever css is already going on in the project.
